module/VuFind/src/VuFind/ILS/Driver/PAIA.php

```diff
@@ -2171,4 +2185,120 @@ protected function paiaDeleteRequest($file, $access_token = null)
         // return TRUE on success
         return true;
     }
+
+    /**
+     * Check whether the patron has any blocks on their account.
+     *
+     * @param array $patron Patron data from patronLogin().
+     *
+     * @return mixed A boolean false if no blocks are in place and an array
+     * of block reasons if blocks are in place
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function getAccountBlocks($patron)
+    {
+        $blocks = [];
+        if (
+            !$this->paiaCheckScope(self::SCOPE_UPDATE_PATRON)
+            && !in_array(self::SCOPE_UPDATE_PATRON, $this->mutedScopes)
+        ) {
+            array_push(
```

Comment:
The syntax `$blocks[] = 'no_update_patron_scope';` might make this a bit more concise.